src/providers/codebuild.ts

```diff
@@ -26,11 +26,11 @@ export interface CodeBuildRunnerProps extends RunnerProviderProps {
   readonly imageBuilder?: IImageBuilder;
 
   /**
-   * GitHub Actions label used for this provider.
+   * GitHub Actions label used for this provider. If multiple labels are specific, a workflow must specify all of them for this provider to be used.
```

Comment:
We use these labels to choose which provider will spawn a runner. If all the labels here are not specified, we won't be able to start that runner. Maybe something like this will be clearer:

> These labels are used to identify which provider should spawn a new on-demand runner. Every job sends a webhook with the labels it's looking for based on `runs-on`. We use match the labels from the webhook with the labels specified here. If all the labels specified here are present in the job's labels, this provider will be chosen and spawn a new runner.

Reply:
@pharindoko does the new text make sense to you?

Reply:
Yes I'm fine with it :)